operator/data/profiles/preview.yaml

```diff
@@ -0,0 +1,6 @@
+# The preview profile contains features that are experimental.
+# This is intended to explore new features coming to Istio.
+# Stability, security, and performance are not guaranteed - use at your own risk.
```

Comment:
Might consider starting this yaml with this warning rather than concluding the comment with it.